src/uct/api/uct.h

```diff
@@ -357,6 +357,35 @@ enum uct_cb_flags {
 };
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Pending request flags.
+ *
+ * List of flags for a pending request @ref uct_pending_req::flags.
+ * A pending request must have either the SYNC or ASYNC flag set.
+ */
+enum uct_pending_req_flags {
+    UCT_PENDING_REQ_FLAG_SYNC  = UCS_BIT(1), /**< Request is always progressed
```

Comment:
didn't we say there will be one flag (ASYNC) and not 2?

Reply:
Also why not starting at `UCS_BIT(0)`?

Reply:
> didn't we say there will be one flag (ASYNC) and not 2?

I thought that relevant only if we merge it with `uct_cb_flags`...

> Also why not starting at UCS_BIT(0)?

By analogy with `uct_cb_flags`

Reply:
what is the point to start with the second bit there?